packages/backend/src/types/UserToken.ts

```diff
@@ -0,0 +1,6 @@
+export interface UserToken {
+  iat: number
+  exp: number
+  userId: string
+  userPermanentId: string // TODO add validation this exists
```

Comment:
I'm assuming this is just imported, but we shouldn't have a distinction of userIds and permanentIds in Rafiki

Reply:
Good catch, will remove :+1: